panel/widgets/input.py

```diff
@@ -162,15 +162,15 @@ def save(self, filename):
                 "filename was given. Please provide a list of filenames or "
                 "file-like objects."
             )
-        elif not isinstance(filename, list) and isinstance(value, list):
+        if not isinstance(filename, list) and isinstance(value, list):
```

Comment:
You're right. 

I changed all `elif` statements to `if` because I see them as separate checks and not dependent on the previous statement. 

I can change it back to `elif` if you want.

Reply:
Thanks!